supreme_court_predictions/models/xg_boost.py

```diff
@@ -70,24 +85,28 @@ def create(self, df):
         """
         self.print("Creating bag of words")
 
-        vectorizer = CountVectorizer(analyzer="word", max_features=5000)
+        vectorizer = CountVectorizer(
+            analyzer="word", max_features=self.max_features
+        )
         vectorize_document = df.loc[:, "tokens"].apply(" ".join)
         bag_of_words_x = vectorizer.fit_transform(vectorize_document)
         bag_of_words_y = df.loc[:, "win_side"]
 
         X_train, X_test, y_train, y_test = train_test_split(
             bag_of_words_x,
             bag_of_words_y,
-            test_size=0.20,
+            test_size=self.test_size,
```

Comment:
Here I am using the constant declared as attribute in the class, or is there something I am missing?

Reply:
It's for the random seed.